extmod/moductypes.c

```diff
@@ -391,7 +399,11 @@ STATIC mp_obj_t uctypes_struct_attr_op(mp_obj_t self_in, qstr attr, mp_obj_t set
     mp_obj_uctypes_struct_t *self = MP_OBJ_TO_PTR(self_in);
 
     // TODO: Support at least OrderedDict in addition
```

Comment:
I guess this TODO comment should be deleted as part of this commit?

Reply:
> I guess this TODO comment should be deleted as part of this commit?

Yes, fixed.